pom.xml

```diff
@@ -78,11 +84,31 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
```

Comment:
Why do we need to pull all these and not get them transitively via jdbc-thin?

Reply:
For `com.amazonaws:aws-java-sdk-core`, I got `Used undeclared dependencies found` error from `maven-dependency-plugin` if it's not explicitly specified in dependencies.

Reply:
For other dependencies like ` com.google.code.gson`, `Failed while enforcing releasability.` errors is raised by `maven-enforcer-plugin` if it's not specified in dependencies.